libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/room/RoomSummary.kt

```diff
@@ -35,7 +36,7 @@ data class RoomSummaryDetails(
     val name: String,
     val isDirect: Boolean,
     val avatarURLString: String?,
-    val lastMessage: CharSequence?,
+    val lastMessage: RoomMessage?,
```

Comment:
I'd directly expose the EventTimelineItem and remove the RoomMessage class as the TimelineItem holds all the data.